packages/camltc/camltc.0.9.4/url

```diff
@@ -0,0 +1,2 @@
+git: "https://github.com/toolslive/camltc.git#camltc-0.9.4"
```

Comment:
Do you prefer a commit hash, or do you want an url that points to an archive combined with a checkum ?